batch-submitter/Dockerfile

```diff
@@ -1,7 +1,8 @@
-FROM golang:1.18.0-alpine3.15 as builder
+FROM golang:1.19-alpine3.17 as builder
 
+RUN sed -i 's/dl-cdn.alpinelinux.org/mirrors.tuna.tsinghua.edu.cn/g' /etc/apk/repositories
 RUN apk add --no-cache make gcc musl-dev linux-headers git jq bash
-
+RUN go env -w GOPROXY=https://goproxy.cn
```

Comment:
this is not needed in CICD. remove it ?

Reply:
For some containers it will be faster to configure the mainland proxy, I will introduce a new file called dockerfile.local